### ISSUE TITLE
[docs] scavenge + setup guide for noobs

### ISSUE THREAD

**Issue by bionicles** [hsoft/collapseos#8]
Hi,

It would be interesting to have a guide for where to scavenge these z80s (I’m a biologist and never heard of them before today) and also set them up install them

It would be cool if they could somehow make a network or software defined radio. That would be a killer app for sure

Will keep an eye on this project, thank you,

 Bion

**Comment by bionicles** [comment]
Do you think folks might buy a kit?

**Comment by hsoft** [comment]
[RC2014](https://rc2014.co.uk) is a very good way to start

**Comment by gtoravan** [comment]
A morse code radio app for the OS will be dope.

**Comment by jodavaho** [comment]
@bionicles I'd buy a kit. But a "guide" should come with three things:

1. A working example of hardware to prototype against (like the RC2014 perhaps)
2. A guide on finding the parts required to assemble (1), 
3. A guide to boostrap everything.

On 3: You can have a kit, and you can have parts to re-build a kit, but there's no way to easily program that first salvaged part. You'll have to structure the guide around setting up the kit to program other, salvaged kits.

**Comment by bastetfurry** [comment]
> but there's no way to easily program that first salvaged part.

Well, you can cough up a manual EEPROM programmer without much hassle and then adapt that to your system later after you bootstrapped it.

**Comment by f4grx** [comment]
EPROM programming is:
-set of switches for address and data lines
-one NE555 for high voltage generation (boost generator)
-one NE555 for programming pulse

A Z80 system could be used as programmer.

**Comment by keithstellyes** [comment]
> Well, you can cough up a manual EEPROM programmer without much hassle and then adapt that to your system later after you bootstrapped it.

The way I picture it, you'd have to use something like dip-switches, a button and a de-bouncer to bootstrap the system if you don't have a computer available, I've seen a lot of homebrew projects do this at a very initial stage, although they always end up just using an Arduino or other device to program it. Is this what you're saying?

I do think there is a whole discussion to be had about boot-strapping the OS without a computer, such as a pared-down version that would be easier to manually program (or to have self-error checking), but that also seems really difficult. Be an interesting engineering problem for sure

**Comment by fadafez** [comment]
Hi, 
It's my first post in this project. I'm really fascinated by it and I would like to give my contribuition by writing code or creating electronic devices, but also with a "couple" of arguments that you will find written below.

There are some points, in my opinion, that are crucial, to make the difference in the success of this initiative.

At the moment I haven't digged the post list, there's already another discussion talking about how the OS will be diffused?

Anyway, as @bionicles pointed out, a radio connection it's foundamental, it can be used for communication, updates delivery and so on.
I'm a programmer, but I've always had a great passion for electronics. In the last few months I have repaired some old radios.

I didn't know, but I found that these old radios were able to receive radio stations located all over the world.
You can see it with your own eyes: on the front panel, where the frequency is indicated, often the name of a city is indicated on old vacuum tube radios. Each big city was associated with a specific frequency.

We are talking about short, medium and long waves, respectively named SW, MW, LW (in AM modulation).
So we already have the technology to receive long range communications, and it's integrated in consumer electronics (yes, not recent devices, almost here in Europe)

Said that, I would like to join the project...how do I do? :-)
Also, there are some topics I would like to discuss with you, it's this the place where I can do that, or there's another more indicated? 

Bye,
Federico Ricchiuto

**Comment by f4grx** [comment]
Speaking of the basics, such a system might need 74xx octal bus drivers.
If you were to design a board, would you use

74xx244: pin are mangled but chip seems common
74xx541: pins are in row order, but chip seems more uncommon.

Any feeling about that?

**Comment by hsoft** [comment]
Sorry, until today, I lacked time to process all inputs to the project, I had to prioritize.

This is an interesting subject, a fundamental one: what kind of information can we put together to maximize the chances for a given post-collapse community to bootstrap itself using the tools at hand?

Of course, the documentation as it is is grossly inadequate, it needs improvement.

I think that the pull request model of github is too rigid to spur collaboration, but at the same time, I also think that public wikis lack focus.

What about brainstorming in a public wiki and then transfer this to `doc/` through the PR model when we think we have something?

I invite you to continue the discussion in #32 to avoid duplicating contents. Closing.
